specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/networkWatcher.json

```diff
@@ -3339,12 +3339,29 @@
           "type": "integer",
           "default": 60,
           "description": "Monitoring interval in seconds."
-        }
+        },
+        "endpoints": {
+          "type": "array",
+          "description": "List of connection monitor endpoints.",
+          "items": {
+            "$ref": "#/definitions/ConnectionMonitorEndpoint"
+          }
+        },
+        "testGroups": {
+          "type": "array",
+          "description": "List of connection monitor tests.",
+          "items": {
+            "$ref": "#/definitions/ConnectionMonitorTestGroup"
+          }
+        }, 
```

Comment:
Please remove trailing space

Reply:
didn't get it

Reply:
Extra space after comma
![image](https://user-images.githubusercontent.com/8080186/59248646-b5050c00-8c2b-11e9-8de3-746807fe30dd.png)